containers/pom.xml

```diff
@@ -84,6 +75,20 @@
 
   <profiles>
     <profile>
+      <id>default</id>
+      <activation>
+        <activeByDefault>true</activeByDefault>
```

Comment:
Do not use activeByDefault, it does not work as expected. There is even a feature request to enforce this via the checkstyle plugin.